packages/mobile/src/recipients/recipient.test.ts

```diff
@@ -1,19 +1,37 @@
+import { MinimalContact } from 'react-native-contacts'
 import { contactsToRecipients, sortRecipients } from 'src/recipients/recipient'
-import { mockContactList, mockRecipient, mockRecipient2, mockRecipient3 } from 'test/values'
+import { mockRecipient, mockRecipient2, mockRecipient3 } from 'test/values'
 
 describe('contactsToRecipients', () => {
   it('returns a recipient per phone number', () => {
     const countryCode = '+1'
-    const recipients = contactsToRecipients(mockContactList, countryCode)
+    const mockContacts: MinimalContact[] = [
```

Comment:
FWIW, it was hard to update the test or understand expected behavior when the salient values were tucked away in test/values, so I moved them there.